blueman/plugins/applet/NetUsage.py

```diff
@@ -207,8 +205,8 @@ def update_time(self):
             delta = datetime.datetime.now() - self.datetime
 
             d = ngettext("day", "days", delta.days)
-            h = ngettext("hour", "hours", delta.seconds / 3600)
-            m = ngettext("minute", "minutes", delta.seconds % 3600 / 60)
+            h = ngettext("hour", "hours", delta.seconds // 3600)
+            m = ngettext("minute", "minutes", delta.seconds % 3600 // 60)
```

Comment:
You are changing to floor division here, this needs imo its own commit with explanation.